scripts/check_env_passthrough.py

```
@@ -100,8 +102,33 @@ def vars_passed(compose_files):
+    "DATABASE_URL",
+    "DATABASE_USERNAME",
+    "DATABASE_PASSWORD",
+    "MINIO_ENDPOINT",
+    "MINIO_ACCESS_KEY",
+    "MINIO_SECRET_KEY",
+    "MINIO_BUCKET",
+    "TZ",
+}
+
+
+def documented_vars(path="\u002eenv.example"):
```

Ruff (0.16.1)

[warning] Unused function argument: `path`

(ARG001)